src/sql/workbench/api/browser/mainThreadExtensionManagement.ts

```diff
@@ -29,4 +35,28 @@ export class MainThreadExtensionManagement implements MainThreadExtensionManagem
 	public $install(vsixPath: string): Thenable<string> {
 		return this._extensionService.install(URI.file(vsixPath)).then((value: ILocalExtension) => { return undefined; }, (reason: any) => { return reason ? reason.toString() : undefined; });
 	}
+
+	public $showObsoleteExtensionApiUsageNotification(message: string): void {
+		console.warn(message);
+
+		if (this._obsoleteExtensionApiUsageNotificationShown) {
+			return;
+		}
+
+		let enableObsoleteAPINotification = this._configurationService.getValue('workbench')['enableObsoleteApiUsageNotification'];
+		if (enableObsoleteAPINotification !== undefined && !enableObsoleteAPINotification) {
```

Comment:
Why not just have it be neverShowObsoleteApiNotification and set that to true instead? Then you can simply just do

```
if (neverShowObsoleteApiNotification) {
…
```

Not a huge deal but it's easier to think about and is typically how I see config values like this be done